core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -113,6 +114,8 @@
       "kafkastore.ssl.endpoint.identification.algorithm";
   protected static final String KAFKASTORE_CONNECTION_URL_DOC =
       "Zookeeper url for the Kafka cluster";
+  protected static final String KAFKASTORE_BOOTSTRAP_SERVERS_DOC =
+      "A list of Kafka brokers to connect to.";
```

Comment:
You probably want to include more info here about the default functionality and when you might want to use this setting. In a *lot* of cases, the user never has to touch this. Also, will this mean that if I specify this I don't need to specify the `kafkastore.connection.url` setting? If I don't want the schema registry to automatically create the topic and do other similar admin actions, can I omit it? Now that we have settings for both ZK and Kafka, it'll definitely need some explanation.